wsmaster/che-core-api-agent/src/main/resources/agents/scripts/org.eclipse.che.ssh.script.sh

```diff
@@ -92,7 +92,11 @@ else
     exit 1
 fi
 
-ps -fC sshd && exit
+if echo ${LINUX_TYPE} | grep -qi "alpine"; then
+   pidof sshd >/dev/null 2>&1 && exit
```

Comment:
ok 
do we need to use the check on linux version ? or we could check against the availability of the tool ?
if pidof is there use it else use ps else use another tool, etc ?